docs/rendering/modelextensions/rendertypes.md

```diff
@@ -0,0 +1,31 @@
+Render Types
+============
+
+Adding the `render_type` entry at the top level allows specifying the render type this model will render in. If this is not specified, the model will fall back to the render types returned by `ItemBlockRenderTypes#getRenderLayers()`.
+
+This replaces the deprecated method of setting the applicable render type(s) via `ItemBlockRenderTypes#setRenderLayer()` for blocks.
```

Comment:
Probably better as a `!!! note` and provide what version did this. Also mention that this is **only** for blocks and not fluids.

Reply:
Converted to a note, not sure how I can make it more specific to blocks since the sentence ends as such.